src/inheritance/Speaker.java

```diff
@@ -1,5 +1,5 @@
 package inheritance;
 
-public interface Speaker {
+public interface Speaker { //access modifier <<abstract>> is implied
```

Comment:
abstract is not an access modifier, but you're right in that `abstract` is implied.